docs/user-profile-and-api-key.md

```diff
@@ -0,0 +1,30 @@
+---
+id: user-profile-and-api-key
+title: User Profile & API Key
+---
+The ARGO User Profile page displays important user information including:
+- which email you have logged into the ARGO platform with
+- what your ARGO Member permission are
+- API Key management
+
+To get to your User Profile, click the initial icon in the top right corner and select Profile and Token.  
```

Comment:
click the "Hello, Username" menu item in the top right corner and select the "Profile & Token" option.

(in case their initials don't make sense)

Reply:
how a about `user icon` ?  I feel the directionality is enough to get people to right place...

Reply:
sure...so update to 
"click the user icon in the top right corner and select the "Profile & Token? option.